stages/org.osbuild.systemd.unit

```diff
@@ -60,12 +77,17 @@ def main(tree, options):
     unit = options["unit"]
     dropin_file = options["dropin"]
     cfg = options["config"]
+    systemd_options = options["unit-type"]
```

Comment:
The `unit-type` key is optional, so this should be
```python
unit_type = options.get("unit-type", "system")
```
to handle the default when it's not specified.

Reply:
updated, I assumed that ` "default": "system"` takes care of it.